src/main/java/com/softserve/academy/spaced/repetition/controller/ExceptionHandlerController.java

```diff
@@ -20,12 +20,12 @@
 @ControllerAdvice
 public class ExceptionHandlerController extends ResponseEntityExceptionHandler {
 
-    private static final EnumMap<AccountStatus, ResponseEntity<MessageDTO>> MAP = new EnumMap<AccountStatus, ResponseEntity<MessageDTO>>(AccountStatus.class);
+    private static final EnumMap<AccountStatus, ResponseEntity<MessageDTO>> STATUS = new EnumMap<>(AccountStatus.class);
```

Comment:
I would rename to something like USER_STATUS_ERROR_RESPONSE.

Reply:
ok) thank you